interfaces/HO-Portal/src/app/program/program-payout/program-payout.component.ts

```diff
@@ -232,6 +232,10 @@ export class ProgramPayoutComponent implements OnInit {
       this.exportInstallmentAvailable = false;
       return;
     }
+    if (isNaN(this.exportInstallmentId)) {
+      this.exportInstallmentAvailable = true;
+
```

Comment:
I think we need a `return` here, so we don't do the request to the endpoint with an 'invalid' `exportInstallmentId`.